smithy-kotlin-codegen/src/main/kotlin/software/amazon/smithy/kotlin/codegen/rendering/waiters/KotlinJmespathExpressionVisitor.kt

```diff
@@ -71,11 +71,20 @@ class KotlinJmespathExpressionVisitor(val writer: KotlinWriter) : ExpressionVisi
     }
 
     override fun visitAnd(expression: AndExpression): String {
-        throw CodegenException("AndExpression is unsupported")
+        writer.addImport(RuntimeTypes.Utils.truthiness)
+
+        val left = expression.left
```

Comment:
It's a nit for sure, but these `left`/`right` temporaries don't seem warranted.

Reply:
You're right, these were originally more complicated and can be omitted now.